pkgs/swift2objc/lib/src/transformer/transform.dart

```diff
@@ -15,6 +15,7 @@ import 'transformers/transform_compound.dart';
 import 'transformers/transform_globals.dart';
 
 typedef TransformationMap = Map<Declaration, Declaration>;
+final primitiveWrapperClasses = <Declaration>{};
```

Comment:
Oh, I see. I misunderstood your question before. You were asking about `primitiveWrapperClasses`, not `primitiveWrappers`. I got them mixed up because the names are so similar.

Yeah, it's not good to have this mutable global variable. Try storing these classes in the `transformationMap`?